aiperf/common/enums/metric_enums.py

```diff
@@ -197,12 +189,6 @@ class GenericMetricUnit(BaseMetricUnit):
     USER = _unit("user")
 
 
-class MetricDateTimeUnit(BaseMetricUnit):
-    """Defines the various date time units that can be used for metrics."""
-
-    DATE_TIME = _unit("datetime")
```

Comment:
i removed the datetime unit, as i felt having the values just be left as epoch nanoseconds was easier for an end user to use